internal/handlers/pg/msg_insert.go

```diff
@@ -123,5 +123,5 @@ func (h *Handler) insert(ctx context.Context, tx pgx.Tx, sp *pgdb.SQLParam, doc
 		return common.NewErrorMsg(common.ErrInvalidNamespace, msg)
 	}
 
-	return lazyerrors.Error(err)
+	return common.NewWriteErrorMsg(53, err.Error())
```

Comment:
Not sure how to handle error code here. Technically no test is failing so I assume that's only one code used here. However, I assume that we want to make this code a variable inside ErrorCodes

Reply:
Yeah, this `53` should be a constant in the `internal/handlers/common/error.go` file.